ci/cpu/build.sh

```diff
@@ -23,6 +23,7 @@ cd $WORKSPACE
 # Get latest tag and number of commits since tag
 export GIT_DESCRIBE_TAG=`git describe --abbrev=0 --tags`
 export GIT_DESCRIBE_NUMBER=`git rev-list ${GIT_DESCRIBE_TAG}..HEAD --count`
+export GIT_COMMIT_SHORT=$(git rev-parse --short $GIT_COMMIT)
```

Comment:
there is a built in conda way to do this: using the var `GIT_DESCRIBE_HASH` if we setup the source to be `git_url` instead of `path`